yarn.lock

```diff
@@ -5236,6 +5238,11 @@ unbox-primitive@^1.0.2:
     has-symbols "^1.0.3"
     which-boxed-primitive "^1.0.2"
 
+undici-types@~5.25.1:
```

Comment:
this version fixes missing the native `fetch` global.
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66824
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/60924

Reply:
`@types/node-fetch` is no longer needed